core/variable_map.js

```diff
@@ -74,10 +74,16 @@ Blockly.VariableMap.prototype.renameVariable = function(variable, newName) {
   Blockly.Events.setGroup(true);
   try {
     // The IDs may match if the rename is a simple case change (name1 -> Name1).
```

Comment:
Delete this comment, since it's no longer relevant.

Reply:
Done.